sr_data_visualization/uis/handescene.rviz

```diff
@@ -0,0 +1,208 @@
+Panels:
```

Comment:
now that we are not using the rviz visualizers, not sure if we should leave these there

Reply:
Removed